sidebars.js

```diff
@@ -194,6 +194,11 @@ const sidebars = {
         },
       ],
     },
+    {
+      type: "doc",
+      label: "Integration guides",
+      id: "integration-guides/index",
+    },
```

Comment:
Because the h1 and label are the same and we're not adding additional data here, we can cut this down to one line:

```suggestion
    "integration-guides/index",
```

(but if I'm missing context for why it's expanded into an object, please just let me know!)